dataverse-webapp/src/main/webapp/file.xhtml

```diff
@@ -194,6 +194,10 @@
                             </div>
                         </div>
                     </div>
+                    <a id="currentDownloadRequestLink" href="#" target="_blank" rel="noopener noreferrer" download="" style="display: none;">
+                        download requested files
```

Comment:
Shouldn't be taken from bundle?

Reply:
It's fully hidden from the user and it is intended to be this way all the time. It doesn't really matters if it is translated or not - so I decided not to add this to bundle to not create extra overhead in for example translation process (thinking about where it is used, what it does, etc.). I guess if we really want to add something from bundle then probably the best key would be: download=Download (as it is used either way in many places). What do you think?

Reply:
If it's hidden then it's ok. Though I wonder how will behave screen readers?

Reply:
screen readers will omit things that are not visible (for example with `display: none;`).
Here is useful link https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/ that confirms this